tools/scripts-core/scripts/test-ng.js

```diff
@@ -1,13 +1,13 @@
-const spawn = require('cross-spawn');
-const { resolveBin } = require('../utils/path-resolver');
-const { getPreset } = require('../utils/preset');
+/* eslint-disable import/extensions */
```

Comment:
Other option could be to move it to a subfolder like we have for utils.
I don't know if utils could be a good option or if we need to create a new one but it could be good to keep in the index only main script (this comment is also applicable for lint)